src/transformers/models/blip_2/modeling_blip_2.py

```
@@ -1870,6 +1876,7 @@ def forward(
 class Blip2ForConditionalGeneration(Blip2PreTrainedModel, GenerationMixin):
     config_class = Blip2Config
     main_input_name = "pixel_values"
+    _supports_sdpa = False
```

hmm, I think SDPA tests should be oke since we have a test designed for multimodals

---

I found that when loading a config with `from_pretrained` all kwargs are set in the config without hard checks for attention implementation. I remember now that it is expected to fail since the user explicitly asks for SDPA, instead of setting `None` to fallback to "eager" when it cannot dispatch SDPA

Let's for now skip this test in BLIP and add a TODO, so BLIP still supports SDPA implicitly when dispatched from `None`.  I will try to get back and work on attention flags soon

---

It's ok, it's just one test and we have fixed quite many tests :-). I will keep as it is (i.e. not to skip the test and not to do `_supports_sdpa = False`) and merge.

---

FYI, `Blip2ForConditionalGenerationDecoderOnlyTest` 

> tests/models/blip_2/test_modeling_blip_2.py::Blip2ForConditionalGenerationDecoderOnlyTest::test_eager_matches_sdpa_generate

also gives the same error since June 07